pi_piper-sysfs.gemspec

```diff
@@ -0,0 +1,32 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'pi_piper/sysfs/version'
```

Comment:
Just let me finish, the implementation of the drivers I'm currently writing. Then you 'll be able to rewrite as you may wish.

By the way the two last points of "Current approach" aren't true. current implementation allows to have methods or variables in the Driver superclass accessible by other drivers. they are just not accessible  by bundler or rake (which doesn't need it anyway)